### PR TITLE
moderation: Create GitHub rejection issue for unverified apps

### PR DESCRIPTION
**Pull request by barthalion**

### CODE REVIEW

File: backend/app/emails.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ class EmailInfo(BaseModel):
     template_data: dict[str, Any]
     # Only works when app_id is set and email is not user specific
     inform_moderators: bool = False
+    inform_only_moderators: bool = False
```
Comment (razzeee) [inline backend/app/emails.py:49]: Do we really want the four cases this generates in combination with `inform_moderators`

I would think we only want three?
- Users
- Moderators
- All

would  `inform_moderators = false` and `inform_only_moderators = true` even make sense?

File: backend/app/moderation.py [hunk 2]
```diff
@@ -459,17 +500,28 @@ def submit_review(
...
+    verification_status = get_verification_status(request.appid)
+
+    if request.is_approved:
+        category = EmailCategory.MODERATION_APPROVED
+        subject = f"Change in build #{request.build_id} approved"
+    else:
+        category = EmailCategory.MODERATION_REJECTED
+        subject = f"Change in build #{request.build_id} rejected"
+
+    if not verification_status.verified:
```
Comment (razzeee) [inline backend/app/moderation.py:513]: I'm not sure why this is the deciding factor here, as an app can still have developers with emails assigned, even if it's unverified.

File: backend/app/emails.py (hunk 1 above)
Comment (barthalion) [reply backend/app/emails.py:49]: We should probably rework this to always send specific types of messages to moderators directly in the e-mail function but I will be worrying about this later

File: backend/app/moderation.py (hunk 2 above)
Comment (barthalion) [reply backend/app/moderation.py:513]: Marking as resolved as per what I sent on Matrix:

>people who submitted an app using github are more likely to never log in to the website
>so we don't have their e-mails, and they won't know a build was rejected
>while for direct upload, you have to log in to verify the ID before uploading anything

**Pull request merged**